example/matrixAddWithMdspan/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright 2023 Erik Zenker, Benjamin Worpitz, Jan Stephan
```

Comment:
Needs to be changed

Reply:
ok changed to 2024 and SPDX-License-Identifier: MPL-2.0

Reply:
Don't forget the author name

Reply:
done, thanks.